### DISCUSSION TITLE
Help！！！

### DISCUSSION THREAD

**Discussion by xiaoli007** [seaweedfs/seaweedfs#4334] [General]
My environment is a cross-regional data center (4 nodes, 4 volumes, 3master). 2 master in Shenzhen, 1 master in Shanghai), the master replica policy is 110, Shanghai users download files will access data from Shenzhen nodes, which leads to slow download speed, because of the 110 strategy, I want to ask Shanghai users how to set up to access the copy of the Shanghai node to download when requesting to download files. Will the file download speed be much faster?

**Comment by kmlebedev** [comment]
Do you see this in the data transfer metrics from volume servers?  Need more details in your installation.  When starting filer'a and volum' servers with specified datacenter parameters, filer should prefer to download a file from its datacenter
